fe/fe-core/src/main/java/com/starrocks/system/ComputeNode.java

```diff
@@ -703,7 +704,8 @@ public boolean handleHbResponse(BackendHbResponse hbResponse, boolean isReplay)
                 CoordinatorMonitor.getInstance().addDeadBackend(id);
             }
         }
-        return isChanged;
+        // If heartbeatRetryTimes is changed, replicate this HbResponse to followers as well.
+        return isChanged || oldHeartbeatRetryTimes != this.heartbeatRetryTimes;
```

Comment:
The alive status has been synced to follower, what is the heartbeatRetryTimes in follower used for?

Reply:
the follower will count this number and set node to dead, and then revert the alive status back to true. which is not consistent with the server.

This part of code worth a refactor to make it much clear, but it will be in a separate PR, this PR will give a band-aid to simply fix the issue.